### PR TITLE
Add python3-qt5-bindings rule for RHEL

### PR DESCRIPTION
**Pull request by cottsay**
python3-qt5-devel is part of EPEL: https://apps.fedoraproject.org/packages/python3-qt5-devel

### CODE REVIEW

**Comment by cottsay** [comment]
Actually, hold off on this. The sip executable isn't working right.

**Comment by cottsay** [comment]
This is now the correct rule, but there is a packaging bug stemming from the non-standard `sip` executable that needs to be addressed before this can be used: https://bugzilla.redhat.com/1742219

The `python_qt_binding` package also needs to be updated to account for the non-standard `sip` executable.

**Comment by tfoote** [comment]
@cottsay should we merge this as is and wait for the upstream bug before using it then?

**Comment by cottsay** [comment]
The rule is correct as-is - this is the package that satisfies the dependency. I think it can be merged, and I'll continue to track the bug upstream and keep the affected packages out of the CentOS nightlies until it is resolved.

tl;dr - I think it can be merged whenever.

**Review by nuclearsandwich — APPROVED** [review]

**Review by clalancette — APPROVED** [review]
